### PR TITLE
Tickets/dm 38521

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: ts_conda_build/conda/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package:
   name: ts-conda-build
-  version: 0.3.11
+  version: 0.3.12
```
Comment (tribeiro) [inline ts_conda_build/conda/meta.yaml:3]: You probably don't need to change the version for this change do you?

File: ts_develop/conda/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package:
   name: ts-develop
-  version: 0.3.19
+  version: 0.3.20
```
Comment (tribeiro) [inline ts_develop/conda/meta.yaml:3]: Same as in ts-conda-build, you probably don't need to change the version

**Review by tribeiro — APPROVED** [review]
Just one comment in both cases, I don't think we need to change the version.

File: ts_conda_build/conda/meta.yaml (hunk 1 above)
Comment (wvreeven) [reply ts_conda_build/conda/meta.yaml]: Agreed, reverted!

File: ts_develop/conda/meta.yaml (hunk 2 above)
Comment (wvreeven) [reply ts_develop/conda/meta.yaml]: Agreed, reverted!

**Pull request merged**